stylus/settings/palette.styl

```diff
@@ -225,4 +225,13 @@ html, .CozyTheme--normal
     --navTextColor var(--slateGrey)
     --navTextActiveColor var(--slateGrey)
     --navTextHoverColor var(--charcoalGrey)
+
+    --alertErrorColor #fff
+    --alertErrorBackgroundColor var(--errorColor)
+
+    --alertSuccessColor #fff
+    --alertSuccessBackgroundColor var(--validColor)
+
+    --alertInfoColor #fff
+    --alertInfoBackgroundColor var(--slateGrey)
```

Comment:
We don't use semantic color here ? (for example --infoColor)

Reply:
In the theme, we are already "high" enough that we can use absolute colors IMHO.